tools/run-app.js

```diff
@@ -708,7 +715,11 @@ _.extend(AppRunner.prototype, {
 
         // Notify the server that new client assets have been added to the
         // build.
-        appProcess.proc.kill('SIGUSR2');
+        if (process.platform === "win32") {
+          appProcess.proc.kill();
+        } else {
+          appProcess.proc.kill('SIGUSR2');
```

Comment:
we need to change how we tell child process "hey, your client program might have changed", @estark37, did you work on this at some point?